tests/unit/parallel/algorithms/any_of.cpp

```diff
@@ -387,12 +387,11 @@ void any_of_bad_alloc_test()
 ///////////////////////////////////////////////////////////////////////////////
 int hpx_main(boost::program_options::variables_map& vm)
 {
-    unsigned int seed = (unsigned int)std::time(nullptr);
+    unsigned int seed = std::random_device{}();
     if (vm.count("seed"))
         seed = vm["seed"].as<unsigned int>();
 
     std::cout << "using seed: " << seed << std::endl;
-    std::srand(seed);
```

Comment:
@Anushi1998 Thanks for updating the file. Would you mind doing this one as well?

Reply:
Before doing that, please would you clarify me what purpose `srand` serves if there is no call to `std::rand`.Have a look at this [discussion](https://github.com/STEllAR-GROUP/hpx/pull/3204#discussion_r171903143) too.

Reply:
Right, good point. But if the test doesn't use any random numbers the seed is also not needed. As far as I know we don't generally add the seed option if the test doesn't require random numbers, so I would remove it completely, but I haven't been around for too long.

@hkaiser Did you mean to use `rand` somewhere in the test? If not, do you prefer to keep the seed option for consistency even if it's not used, or could it be removed?

Reply:
It has no purpose, please remove it.

Reply:
It has no purpose, please remove it.

Reply:
Sure, I will do it :)